brave/brave-core
# Updated default search providers in KR

Resolves https://github.com/brave/brave-browser/issues/18855

Desktop & android will have Brave Search, Daum, Naver and Google as a default search provider.

### **Desktop search engine settings page**
<img width="663" alt="image" src="https://user-images.githubusercontent.com/6786187/212000590-119b929b-1919-4198-872d-7cb8cabfd4c7.png">


### **Android onboarding page**
<img width="474" src="https://user-images.githubusercontent.com/6786187/211995474-926b1632-5625-4d1b-894f-3c06f9f19822.png">

### **Android search engine settings page**
<img width="474" src="https://user-images.githubusercontent.com/6786187/211996030-446fbe5d-b309-4db4-abfe-4e965b7e46b9.png">

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Set Region settings as South Korea
2. Launch Brave with fresh profile
3. Check search engine settings has SK specific provider list - Brave, Naver, Daum and Google
4. Check Naver is set as an initial default provider
